### ISSUE TITLE
[V1] QTabs positioning error on QToolbar

### ISSUE THREAD

**Issue by digiproductmarketer** [quasarframework/quasar#4209]
I’m having a bit of an issue trying to centralise my QTabs on a QToolbar

Using the QSpace to do it seems to force the left/right arrows to appear too soon.

In the Codepen below …

The first example shows the issue.

The second example is the same as the first example but with the QSpace removed … and the arrows no longer appear …

The third example is the same as the first example but with one less tab … just to show that QSpace does actually do what I wanted

https://codepen.io/david-watson-the-encoder/pen/arWxzY

I got a few suggestings via a thread on the forum 

https://forum.quasar-framework.org/topic/3625/qtabs-positioning-error-on-qtoolbar

But, those suggestions only made the situation worse.

As a workaround, I just allowed my QTabs to justify right as in example 2 in the Codepen ...

**Comment by rstoenescu** [comment]
Hi,

When using QToolbarTitle or QSpace (these two components try to expand as much as they can horizontally), you should specify `shrink` prop on QTabs. Otherwise, QTabs is configured to also spread horizontally, so you end up with 2 or 3 components trying to do the same.

Also, please remember that when using UMD (and codepen uses just that), you shouldn't use self-closing tags.

https://codepen.io/rstoenescu/pen/jopaBj

Please feel free to reopen if I missed anything.

**Comment by digiproductmarketer** [comment]
Hi Razvan,

Sorry I had to re-open this ticket because I'd already tried SHRINK ... and I just went and tried it again to confirm ... that doesn't help.

I also tried shrink on the QToolbarTitle ... but that causes an infinite recalculating loop ...

BUT .. then I tried your codepen ... and it worked ...

So, I couldn't understand it ... until I realised that I was running your codepen on my laptop that is using Windows 10 ... and I was running my codepen on my laptop that runs Windows 7.

So, I tried running your codepen on my Windows 7 laptop ... and your codepen also fails under Windows 7.

Tried both your codepen and my codepen on OS/X High Sierra and both Safari and Chrome and they both work there.

Not sure if you want to persue this further as I am not really bothered about getting it running on Windows 7 for myself ... just so happens I have a development machine that has to remain on Windows 7 and was testing there.


So, it's actually an issue related to Windows 7 ...
